src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/LongDescriptionValidator.cs

```diff
@@ -9,8 +9,11 @@ public class LongDescriptionValidator : AbstractValidator<string>
 
         public LongDescriptionValidator()
         {
-            RuleFor(x => x)
-                .MatchesAllowedHtmlFreeTextCharacters(LongDescriptionShouldNotIncludeSpecialCharacters, LongDescriptionShouldNotIncludeBlacklistedHtmlElements, PropertyName);
+            RuleFor(x => x).MatchesAllowedHtmlFreeTextCharacters(ErrorCodes.CreateApprenticeship.LongDescriptionShouldNotIncludeSpecialCharacters,
+                ErrorMessages.CreateApprenticeship.LongDescriptionShouldNotIncludeSpecialCharacters,
```

Comment:
If the message is going to be the same, perhaps we can define it near the extension so we don't have to pass the message every time we use the extension method.

Reply:
Good call. Will use a generic message that uses the propertyName